test/dynamo/test_exc.py

```diff
@@ -251,13 +251,13 @@ def fn(x, shape):
 
 Model:
   ==> L['shape'][0]: 0
-  ==> L['shape'][1]: 1
-  ==> L['shape'][2]: 1
+  ==> L['shape'][1]: 0
```

Comment:
This looks suspicious.

Reply:
It's not intrinsically suspicious; there are many models Z3 can find and it's not required for it to always produce the same result.